dev-python/matplotlib-venn/matplotlib-venn-9999.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 git-r3
+
+DESCRIPTION="Routines for plotting area-weighted two- and three-circle venn diagrams"
+HOMEPAGE="http://pysurfer.github.com"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/konstantint/matplotlib-venn"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS=""
+IUSE="test"
+
+RDEPEND="
+	dev-python/matplotlib[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]"
+DEPEND="test? (
+		${RDEPEND}
+		dev-python/pytest[${PYTHON_USEDEP}]
+	)"
+
+python_test() {
+	distutils_install_for_testing
```

Comment:
You mean the test phase altogether? Not really, I would be happy to not provide the test feature, especially since it likes to [fail if you happen to be running X](https://github.com/konstantint/matplotlib-venn/issues/21).